source/reference/kubectl-minio-plugin/kubectl-minio-tenant-expand.rst

```diff
@@ -54,15 +54,19 @@ Syntax
 
       .. code-block:: shell
 
-         kubectl minio tenant expand            \
-                              TENANT_NAME       \
-                              --capacity        \
-                              --servers         \
-                              --volumes         \
-                              --namespace       \
-                              [--output]        \
-                              [--pool]          \
+         kubectl minio tenant expand                            \
+                              TENANT_NAME                       \
+                              [--output]                        \
+                              [--pool]                          \
                               [--storage-class]
+                              --capacity                        \
```

Comment:
And again here, I think the required flags should be at the top of the list with the optional flags below them.